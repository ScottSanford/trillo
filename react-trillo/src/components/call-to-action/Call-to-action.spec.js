import React from 'react'
import { shallow } from 'enzyme'
import CallToAction from './Call-to-action'

describe('Call to Action Component', () => {

	it('should match the snapshopt', () => {

		const wrapper = shallow(<CallToAction />)
		expect(wrapper).toMatchSnapshot()
	})
})