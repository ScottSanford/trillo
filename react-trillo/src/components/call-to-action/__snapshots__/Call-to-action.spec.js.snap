// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Call to Action Component should match the snapshopt 1`] = `
<div
  className="cta"
>
  <h2
    className="cta__book-now"
  >
    Good news! We have 4 free rooms for your selected dates!
  </h2>
  <button
    className="btn"
  >
    <span
      className="btn__visible"
    >
      Book now
    </span>
    <span
      className="btn__invisible"
    >
      Only 4 rooms left
    </span>
  </button>
</div>
`;
